scaffolds/python/README.md

```diff
@@ -0,0 +1,24 @@
+# Python Scaffold for Code Retreat
+
+## Install Python tools
+
+The easiest option is to install the [Anaconda](https://www.continuum.io/downloads) distribution, which includes Python plus many popular libraries. Prefer using Python 3.x.
+
+## Run the tests
+
+From `python/src/`, run:
```

Comment:
please consider `python` as your root as the other project. So you can replace it with `src/`.